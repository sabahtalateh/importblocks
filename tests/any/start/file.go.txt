package test1

import(
	"a/b/c2"
	"os"
	"x/y/z2"
	"a/b/c1"
	"x/y/z1"
)
