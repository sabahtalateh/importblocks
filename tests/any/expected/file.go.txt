package test1

import (
	"a/b/c1"
	"a/b/c2"

	"x/y/z1"
	"x/y/z2"

	"os"
)
